proto/text_tools.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package text_tools;
+option go_package = "github.com/ekkinox/grpc-demo/proto";
+option php_namespace = "App\\TextTools";
+option php_metadata_namespace = "App\\GPBMetadata";
+
+import "google/api/annotations.proto";
+
+enum Transformer {
+  TRANSFORMER_UPPERCASE_UNSPECIFIED = 0;
```

Comment:
haha :) 

imo should be 

```
enum Transformer {
  TRANSFORMER_UNSPECIFIED = 0;
  TRANSFORMER_UPPERCASE = 1;
  TRANSFORMER_LOWERCASE = 2;
}
```

I took a shortcut in the demo and skipped the unspecified enum, but imo should be this way.

can you fix it / regen the stubs / fix the go and php code ?